examples/default.yaml

```diff
@@ -67,6 +67,10 @@ mounts:
   # 🔵 This file: true (only for "/tmp/lima")
   writable: true
 
+# Mount type for above  mounts, such as "sshfs" (from sshocker) or "9p" (from virtfs)
+# 🟢 Builtin default: ""
+mountType: null
+
```

Comment:
Can we create “example/experimental-9p.yaml” and test in the CI?

Reply:
That sounds like a good idea! Will add that example - just use default (or ubuntu)